Gruntfile.js

```diff
@@ -23,6 +35,60 @@ module.exports = function(grunt) {
             });
     });
 
+    grunt.registerTask("hugo-server", function() {
+        const done = this.async();
+
+        grunt.log.writeln("Running Hugo server");
+        grunt.util.spawn({
+            cmd: "hugo",
+            args: ["server"],
+            opts: {stdio: 'inherit'}
+        },
```

Comment:
Thought:

```
grunt.registerTask("hugo-server", function() {
  const done = this.async();
  const args = process.argv.slice(3); // fetch given arguments

  grunt.log.writeln("Running Hugo server");
  grunt.util.spawn({
    cmd: "hugo",
    args: ["server", ...args], // pass arguments down
    // ...
```

In this way we could pass any argument to hugo server obviating the need for the extra `hugo-server-fullrender` task and allowing the user to pass any additional arguments they might need.

```
yarn server --disableFastRender --veryVerboseOrSomething
```

Reply:
Thanks @jamesdphillips , I think this is a great idea. I'll give it a shot.